cogent/Setup.hs

```diff
@@ -102,15 +102,28 @@ copyGumHdrs verbosity pkg lbi copy = do
   let hdrsdest = datadir (L.absoluteInstallDirs pkg lbi copy) </> "include"
   putStrLn $ "Installing Gum headers in `" ++ hdrsdest ++ "'..."
   createDirectoryIfMissingVerbose verbosity True hdrsdest
-  installDirectoryContents verbosity "lib" hdrsdest
+  errorMsg <- installDirectoryContents verbosity "lib" hdrsdest
+  case errorMsg of 
+    Nothing -> return ()
+    Just x  -> do putStrLn x; putStrLn "Are you using cabal2.4?"
```

Comment:
What's special in cabal 2.4? Also that thing is usually called `cabal-install-2.4`, to disambiguate from `Cabal-2.4`,